pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
+import subprocess
+import logging
+import os
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+
+
+def test_integration_test(caplog):
+
+    caplog.set_level(logging.INFO)
+
+    ROOT_DIR = os.path.dirname(os.path.abspath("setup.py"))
+
+    for example_script in glob.glob(f"{ROOT_DIR}/examples/calculate_*.py"):
+
+        # run script and check that it doesn't crash
+        subprocess.check_output(f"python {example_script}", shell=True)
```

Comment:
Why `check_output` and not run? Why the unsafe `shell = True`. Also: you should give the arguments as a list of strings, so they can be properly escaped: `sp.run([sys.executable, example_script]) would also make sure, that the file is run with the same python interpreter as the test script.

Reply:
I tried `sp.run([sys.executable, example_script])` and it was not working...

`check_output` is basically `run` but with a further check on the return code so it should be better than just running

`shell=True` could be removed since this script doesn't take any args it is sufficient to use a simple string like it is explained [here](https://docs.python.org/3/library/subprocess.html)